google-cloud-spanner/lib/google/cloud/spanner/commit.rb

```diff
@@ -127,6 +129,8 @@ def upsert table, *rows
         #   See [Data
         #   types](https://cloud.google.com/spanner/docs/data-definition-language#data_types).
         #
+        # @return [Array<Hash>] The array of rows.
```

Comment:
Why is `upsert` returning the rows? I don't think it should be returning anything.